src/Sign.SignatureProviders.TrustedSigning/RSATrustedSigning.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Azure;
+using Azure.CodeSigning;
+using Azure.CodeSigning.Models;
+
+namespace Sign.SignatureProviders.TrustedSigning
+{
+    internal sealed class RSATrustedSigning : RSA
+    {
+        private readonly CertificateProfileClient _client;
+        private readonly string _accountName;
+        private readonly string _certificateProfileName;
+        private readonly X509Certificate2 _publicKey;
```

Comment:
I named it `_publicKey` instead of `_certificate` to make it clear this only contains the public key and not a private key. I can rename it if you really want me to.

Reply:
Okay will do.